### PR TITLE
fee summary updates on transfers

### PR DESCRIPTION
**Pull request by owillborn**
*Issue #:* /bcgov/entity#15295

*Description of changes:*
- Adjusted fee summary on transfers according to spec
- Fee summary hides on cancel

With no transfer type selected:
image

With transfer type selected:
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/composables/fees/FeeSummary.vue [hunk 1]
```diff
@@ -223,6 +236,7 @@ export default defineComponent({
     const localState = reactive({
       feeType: props.setFeeType,
       registrationType: props.setRegistrationType,
+      transferType: props.setTransferType,
```
Comment (cameron-eyds) [inline ppr-ui/src/composables/fees/FeeSummary.vue]: I know this is what has been done elsewhere with props in some older components but unless you are going to mutate this prop value locally, you can use it directly.

File: ppr-ui/src/composables/fees/FeeSummary.vue [hunk 2]
```diff
@@ -352,6 +369,9 @@ export default defineComponent({
     watch(() => props.setRegistrationType, (val: UIRegistrationTypes) => {
       localState.registrationType = val
     })
+    watch(() => props.setTransferType, (val: UITransferTypes) => {
+      localState.transferType = val
+    })
```
Comment (cameron-eyds) [inline ppr-ui/src/composables/fees/FeeSummary.vue]: You won't need this by using the prop directly

File: ppr-ui/src/composables/fees/FeeSummary.vue (hunk 1 above)
Comment (owillborn) [reply ppr-ui/src/composables/fees/FeeSummary.vue]: Gotcha, had it as is just to be consistent. Will change

**Review by cameron-eyds — COMMENTED** [review]
Code looks good! Just need to refactor the prop, what is happening is artificial reactivity, just use the prop directly and let vue take care of the rest :)

File: ppr-ui/src/composables/fees/FeeSummary.vue (hunk 2 above)
Comment (owillborn) [reply ppr-ui/src/composables/fees/FeeSummary.vue]: Changed

**Review by cameron-eyds — APPROVED** [review]
Right on! Less is more, always 😁 
Gj 👍

**Review by doug-lovett — APPROVED** [review]
Looks good.

**Pull request merged**